examples/bar_polar.py

```diff
@@ -0,0 +1,75 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+from datetime import date
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/2011_us_ag_exports.csv"
+df = pd.read_csv(filepath)
+
+state_list = list(df["state"].unique())
+
+chosen_list = ["Alabama", "Alaska", "Arizona"]
+
+app = Dash(__name__)
+
+app.layout = html.Div([
+    html.Div([
+        html.H1(f'Polar Charts of U.S. Agricultural Exports, 2011',
+                style={'textAlign': 'center'})]),
+    html.Div([
+        html.H2('Choose the radius scale (absolute or logarithmic), then choose states:',
+                style={"margin-left": "5em"}),
```

Comment:
Typically, each component would have a label rather than putting all instructions in a heading.

Reply:
I don't understand what that entails in practice. If I remove the html headers, then should I put the instructions in a label? If so, do you have an example in mind?

Reply:
oops, just realized your next comment answers my question. So I've hidden my previous comment.